PC/_wmimodule.cpp

```diff
@@ -308,11 +308,15 @@ static PyMethodDef wmi_functions[] = {
 };
 
 static PyModuleDef wmi_def = {
-    PyModuleDef_HEAD_INIT,
-    "_wmi",
-    NULL,   // doc
-    0,      // m_size
-    wmi_functions
+    .m_base = PyModuleDef_HEAD_INIT,
+    .m_name = "_wmi",
+    .m_doc = _Py_NULL,
+    .m_size = 0,
+    .m_methods = wmi_functions,
+    .m_slots = _Py_NULL,
+    .m_traverse = _Py_NULL,
+    .m_clear = _Py_NULL,
+    .m_free = _Py_NULL,
```

Comment:
C++ doesn't support such initializers.

```suggestion
    /* m_base */ PyModuleDef_HEAD_INIT,
    /* m_name */ "_wmi",
    /* m_doc */ _Py_NULL,
    /* m_size */ 0,
    /* m_methods */ wmi_functions,
    /* m_slots */ _Py_NULL,
    /* m_traverse */ _Py_NULL,
    /* m_clear */ _Py_NULL,
    /* m_free */ _Py_NULL,
```

Reply:
We also can't remove them either, otherwise we get mixed initializer errors, which is the other category or error that we were running into

Reply:
Ugh, getting a fix for C++ gets into C99 mixed designated initializer errors. Getting a C99 fix, gets into C++ not supporting it 😞

Reply:
Fixed now! I ended up removing these initializers as explained below